local-cli/cliEntry.js

```diff
@@ -5,171 +5,151 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
  */
 'use strict';
 
-const bundle = require('./bundle/bundle');
-const childProcess = require('child_process');
+const cli = require('commander');
```

Comment:
just a nit but `cli` is not the best name imho considering how much this term is used